app/views/govuk_publishing_components/components/docs/details.yml

```diff
@@ -0,0 +1,21 @@
+name: Details
+description: Make a page easier to scan by letting users reveal more detailed information only if they need it
+shared_accessibility_criteria:
+  - link
+accessibility_criteria: |
+  The component must:
+
+  * accept focus
+  * be focusable with a keyboard
+  * be usable with a keyboard
+  * be usable with touch
+  * indicate when it has focus
```

Comment:
I would add:
* toggle the visibility of the details element's content when interacted with
* indicate the expanded state when details' content is visible
* indicate the collapsed state when details' content is hidden

Reply:
Thanks, updated.

Reply:
Can't see them in yet. Happy to approve after they're added.

Reply:
They're there 👇